sstables/sstable_writer.hh

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2021 ScyllaDB
+ *
```

Comment:
We now use `Copyright (C) 2021-present ScyllaDB`
And while at it, no need for the extra ` *` line...

Reply:
done